plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/XLanguageServerImpl.java

```diff
@@ -243,9 +245,13 @@ public void setWorkspaceManager(XWorkspaceManager manager) {
 
 		access.addBuildListener(this);
 
+		System.out.println("About to initialize the server with workspace " + baseDir);
```

Comment:
What would be the correct way to do logging here, @mmews-n4 ?

Reply:
Maybe `LOG.info(...);` can be used here?